my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency.html

```diff
@@ -1,76 +1,70 @@
-<div ng-if='!contact.edit || contact.edit == false'>
-    <p>{{ contact.preferredName }}</p>
-    <div ng-repeat="phone in contact.phoneNumbers">
-        <p ng-if='phone.value && phone.type'>
-          <span ng-if="$index === 0">Primary phone: </span>
-          <span>{{ phone.value }} ({{ phone.type }})</span>
-        </p>
+<!-- display emergency contacts -->
+<div ng-if="!contact.edit || contact.edit == false">
+  <p>{{ contact.preferredName }}</p>
+  <div ng-repeat="phone in contact.phoneNumbers">
+    <p ng-if="phone.value && phone.type">
+      <span ng-if="$index === 0">Primary phone: </span>
+      <span>{{ phone.value }} ({{ phone.type }})</span>
+    </p>
+  </div>
+  <p>{{ contact.emails[0].value }}</p>
+  <p ng-if="contact.relationship">Relationship: {{ contact.relationship }}</p>
+  <p ng-if="contact.comments">Comments: {{ contact.comments }}</p>
+  <div layout="row" ng-hide="contact.readOnly" class="buttons-small">
+    <md-button class="md-accent md-raised" ng-click="addEdit()">Add another</md-button>
```

Comment:
In another spot, "Add another" is nuanced for screen readers with a clarifying aria-label (add another *what*?) Seemed like a good move?